NPE in cgeo.geocaching.unifiedmap.AbstractMapFragment.getViewport()

with recent nightlies (at least 1 week) I have many crashs caused by NPE in cgeo.geocaching.unifiedmap.AbstractMapFragment.getViewport()

Those happen in different situations, e.g. editing personal note, opening cache details, going back from cache to map.

Some stacktraces:
java.lang.NullPointerException: Attempt to invoke virtual method 'cgeo.geocaching.location.Viewport cgeo.geocaching.unifiedmap.AbstractMapFragment.getViewport()' on a null object reference
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.refreshListChooser(UnifiedMapActivity.java:571)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.reloadCachesAndWaypoints(UnifiedMapActivity.java:477)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.access$000(UnifiedMapActivity.java:128)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity$1.onReceive(UnifiedMapActivity.java:266)
	at cgeo.geocaching.service.GeocacheChangedBroadcastReceiver.onReceive(GeocacheChangedBroadcastReceiver.java:35)

java.lang.NullPointerException: Attempt to invoke virtual method 'cgeo.geocaching.location.Viewport cgeo.geocaching.unifiedmap.AbstractMapFragment.getViewport()' on a null object reference
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.refreshListChooser(UnifiedMapActivity.java:571)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.$r8$lambda$0tdmED8uHBT03g53if-rjSKFSTs(UnifiedMapActivity.java:0)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity$$ExternalSyntheticLambda30.run(R8$$SyntheticClass:0)
	at cgeo.geocaching.utils.livedata.CollectionLiveData.lambda$observeForNotification$1(CollectionLiveData.java:92)
	at cgeo.geocaching.utils.livedata.CollectionLiveData.$r8$lambda$O5YFKMwJuUNg_jb2gA0FxU212LY(CollectionLiveData.java:0)
	at cgeo.geocaching.utils.livedata.CollectionLiveData$$ExternalSyntheticLambda1.onChanged(R8$$SyntheticClass:0)

Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'cgeo.geocaching.location.Viewport cgeo.geocaching.unifiedmap.AbstractMapFragment.getViewport()' on a null object reference
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.refreshListChooser(UnifiedMapActivity.java:571)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.reloadCachesAndWaypoints(UnifiedMapActivity.java:477)
	at cgeo.geocaching.unifiedmap.UnifiedMapActivity.onResume(UnifiedMapActivity.java:1192)

--- a/main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java
+++ b/main/src/main/java/cgeo/geocaching/unifiedmap/UnifiedMapActivity.java
@@ -571,9 +571,8 @@ public class UnifiedMapActivity extends AbstractNavigationBarMapActivity impleme
     private void refreshListChooser() {
         //try to get count of visible caches
         int visibleCaches = 0;
-        final Viewport viewport = mapFragment.getViewport();
-        if (mapFragment != null && viewport != null && viewModel != null) {
-            visibleCaches = viewModel.caches.readWithResult(viewport::count);
+        if (mapFragment != null && mapFragment.getViewport() != null && viewModel != null) {
+            visibleCaches = viewModel.caches.readWithResult(mapFragment.getViewport()::count);
         }
 
         if (mapType.fromList != 0) {

